### ISSUE TITLE
Is it possible to change the figure size in category_scatterplot?

### ISSUE THREAD

**Issue by hellojinwoo** [rasbt/mlxtend#628]
I am studying machine learning, and visualization has been always thorn in my flesh. I thank this library 'mlxtend' to let me easily draw categorical scatter plots out of pd.DataFrame. However, I have 2 questions regarding it.

* I have thoroughly read your examples from this url: http://rasbt.github.io/mlxtend/user_guide/plotting/category_scatter/#overview

<h3> Q1. Is it possible to change the figure size of the plot? </h3>

`from mlxtend.plotting import category_scatter` does not accept `figsize` as a parameter. How can I change the figure size of the plot?

<h3> Q2. How can I annotate the each point in scatter plots? </h3>
It would be very nice if I can annotate each points in a scatter plot by their label names. Can I do this with mlxtend?

**Comment by rasbt** [comment]
> Q1. Is it possible to change the figure size of the plot?

Yes, that's definitely possible. I think this is among the oldest functions in mlxtend, and the API is not that obvious, but here is a working example (it should probably be added to the documentation, so I am leaving this issue open as a reminder).

Here are some mandatory imports and some data:

```python
import numpy as np
from io import BytesIO
import matplotlib.pyplot as plt
from mlxtend.plotting import category_scatter


csvfile = """1,10.0,8.04
1,10.5,7.30
2,8.3,5.5
2,8.1,5.9
3,3.5,3.5
3,3.8,5.1"""

ary = np.genfromtxt(BytesIO(csvfile.encode()), delimiter=',')
```

Then, you can resize using `set_size_inches`, e.g.,

```python
fig = category_scatter(x=1, y=2, label_col=0, 
                       data=ary, legend_loc='upper left')

fig.set_size_inches(3, 4)
```

> Q2. How can I annotate the each point in scatter plots?

This could be done uses the "`annotate`"  method. For example, using the dataset above, you could do it as follows:

```python

fig = category_scatter(x=1, y=2, label_col=0, 
                       data=ary, legend_loc='upper left')


x = ary[:, 1]
y = ary[:, 2]
l = ['a', 'b', 'c', 'd', 'e', 'f']

for i, txt in enumerate(l):
    fig.axes[0].annotate(txt, (x[i], y[i]))

plt.show()
```

![Unknown](https://user-images.githubusercontent.com/5618407/69080382-7dcc2f80-0a01-11ea-890b-46bc959b6ddc.png)

**Comment by jbbqqf** [comment]
Hi! Triaging older issues this week — this is a ~6-year-old usage question rather than a bug.

To answer both questions against the current `category_scatter`:

- **Q1 (figsize):** `category_scatter` internally calls `fig = plt.figure()` with no `figsize`, see [`mlxtend/plotting/scatter.py:62`](https://github.com/rasbt/mlxtend/blob/master/mlxtend/plotting/scatter.py#L62). The function returns the `fig` object, so the simplest fix in your own code is:
   ```python
   fig = category_scatter(x='x', y='y', label_col='label', data=df)
   fig.set_size_inches(10, 6)
   ```
  Or, set matplotlib's global default before the call: `plt.rcParams['figure.figsize'] = (10, 6)`.

- **Q2 (per-point annotations):** `category_scatter` doesn't expose this, but the returned `fig` gives you the axes (`ax = fig.axes[0]`), so you can call `ax.annotate(label, (x_i, y_i))` in a loop over your rows after the call.

Since both have direct workarounds and there's no follow-up since 2019, this can probably be closed as a usage question. If you'd like a real `figsize=` parameter exposed in the signature, that would be a separate enhancement.

(Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the file/line references above were verified manually against the current `master`.)
